src/citrine/resources/design_workflow.py

```diff
@@ -18,10 +20,44 @@ class DesignWorkflowCollection(Collection[DesignWorkflow]):
     _resource = DesignWorkflow
     _api_version = "v2"
 
-    def __init__(self, project_id: UUID, session: Session, branch_id: UUID = None):
+    def __init__(self,
+                 project_id: UUID,
+                 session: Session,
+                 branch_id: Optional[UUID] = None,
+                 branch_root_id: Optional[UUID] = None,
+                 branch_version: Optional[int] = None):
         self.project_id: UUID = project_id
         self.session: Session = session
-        self.branch_id: UUID = branch_id
+
+        if branch_id:
```

Comment:
```suggestion
        if branch_id is not None:
```

Reply:
This just depends on how aggressively we want to raise the deprecation warning. My thought is that since other things will catch it and fail downstream, I'm not concerned. Plus, if users are constructing their own `DesignWorkflowCollection`, they're already using the SDK in an unorthodox way.